package.json

```diff
@@ -17,9 +17,10 @@
     "format": "prettier --write --cache .",
     "lint": "eslint --cache packages/*/{src,types,__tests__}/** playground/**/__tests__/**/*.ts scripts/**",
     "typecheck": "tsc -p scripts --noEmit && tsc -p playground --noEmit",
-    "test": "run-s test-unit test-serve test-build",
+    "test": "run-s test-unit test-serve test-build test-build-legacy-cjs",
```

Comment:
This is awesome @sodatea! Now that we saw that tests are passing, maybe we could remove this from the main CI tests? We could add a project to vite-ecosystem-ci for example to check it, but making the CI 30% slower because of this seems too much to me. What do you think?

Reply:
Yeah, sure, that makes sense.